### PR TITLE
Add GET permalink to Download Entity Metadata

### PR DESCRIPTION
**Pull request by jmthibault79**
- [ ] **Submitter**: Include the JIRA issue number in the PR description
- [ ] **Submitter**: Check documentation and code comments. Add explanatory PR comments if helpful.
- [ ] **Submitter**: If you changed a URL that is used elsewhere (e.g. in an email), comment about where it is used and ensure the dependent code is updated.
- [ ] **Submitter**: JIRA ticket checks:
  * Acceptance criteria exists and is met
  * Note any changes to implementation from the description
  * To Demo flag is set
  * Release Summary is filled out, if applicable
  * Add notes on how to QA
- [ ] **Submitter**: Update RC_XXX release ticket with any config or environment changes necessary
- [ ] **Submitter**: Update FISMA documentation if changes to:
  * Authentication
  * Authorization
  * Encryption
  * Audit trails
- [ ] **Submitter**: If you're adding new libraries, sign us up to security updates for them
- [ ] **Submitter**: If you're adding new automated UI tests, review the test plan with QA
* Review cycle:
  * LR reviews
  * Rest of team may comment on PR at will
  * **LR assigns to submitter** for feedback fixes
  * Submitter rebases to develop again if necessary
  * Submitter makes further commits. DO NOT SQUASH
  * Submitter updates documentation as needed
  * Submitter **reassigns to LR** for further feedback
- [ ] **TL** sign off
- [ ] **LR** sign off
- [ ] **Product Owner** sign off
- [ ] **Submitter**: Verify all tests go green, including CI tests and automated UI tests.
- [ ] **Submitter**: Squash commits and merge to develop. If adding test code, merge application code and test code at the same time.
- [ ] **Submitter**: Delete branch after merge
- [ ] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
- [ ] **Submitter**: Mark JIRA issue as resolved once this checklist is completed

### CODE REVIEW

**Comment by jmthibault79** [comment]
Current State:

screen shot 2018-06-20 at 5 18 08 pm

This is obviously too ugly for production.  Will consult PO & UI/UX for improvement.

**Comment by jmthibault79** [comment]
Better now:
screen shot 2018-06-21 at 10 31 07 am

File: src/cljs/main/broadfcui/page/workspace/data/tab.cljs [hunk 1]
```diff
@@ -112,30 +112,39 @@
...
+                          :color (:button-primary style/colors) :fontSize "inherit" :fontFamily "inherit"}
+                  :type "submit"
+                  :value (str "Download '" selected-entity-type "' metadata")}]]
+        [:a {:style {:textDecoration "none" :marginLeft "1em"}
+             :href (str (config/api-url-root) "/cookie-authed/workspaces/"
+                        (:namespace workspace-id) "/"
+                        (:name workspace-id) "/entities/" selected-entity-type "/tsv")
+             :onClick #(user/set-access-token-cookie (user/get-access-token))
+             :target "_blank"}
+         "[permalink]"]]))
```
Comment (ghost) [inline src/cljs/main/broadfcui/page/workspace/data/tab.cljs]: May want to set `color: rgb(69, 127, 210)`

**Review by ghost — APPROVED** [review]
One comment, looks good 👍

File: src/cljs/main/broadfcui/page/workspace/data/tab.cljs (hunk 1 above)
Comment (ghost) [reply src/cljs/main/broadfcui/page/workspace/data/tab.cljs]: comment belongs on line 141

Comment (jmthibault79) [reply src/cljs/main/broadfcui/page/workspace/data/tab.cljs]: 👍

**Comment by jmthibault79** [comment]
After Joe's suggestion: 
screen shot 2018-06-21 at 12 33 31 pm

**Review by davidangb — APPROVED** [review]
:+1: for code.

for the UX - I'm fine with the "permalink" text, but consider instead using an icon, with hovertext/tooltip noting it as a permalink?

We have [download](https://fontawesome.com/icons/download?style=solid) already defined in CLJS but could also easily add one of the [link](https://fontawesome.com/icons?d=gallery&q=link) icons.

**Comment by jmthibault79** [comment]
@davidangb @irosenbe New look with "link" icon
screen shot 2018-06-21 at 1 45 09 pm
 and tooltip
screen shot 2018-06-21 at 1 45 21 pm

**Comment by irosenbe** [comment]
Looks good! You have my approval to move forward with the link and text.

**Review by davidangb — APPROVED** [review]
:+1: beautiful!

**Pull request merged**